variables.md

```diff
@@ -65,6 +65,7 @@ INSTALLONLY | boolean | false | Indicates that test suite conducts only installa
 INSTLANG | string | en_US | Installation locale settings.
 IPXE | boolean | false | Indicates ipxe boot.
 ISO_MAXSIZE | integer | | Max size of the iso, used in `installation/isosize.pm`.
+IS_MM_SERVER | boolean | | If set, run server-specific part of the multimachine job
 KEEP_ONLINE_REPOS | boolean | false | openSUSE specific variable, not to replace original repos in the installed system with snapshot mirrors which are not yet published.
```

Comment:
I'd prefer it to be `IS_MULTI_MACHINE_SERVER` (or `IS_MULTIMACHINE_SERVER`), because `MM` is not self explaining (can be memory management or multimedia or whatever).